dom/src/DomSources/MotorcycleDomSource.ts

```diff
@@ -200,9 +200,43 @@ function scopeEventStream(
       return checkElementIsInScope(ev.target as HTMLElement);
     })
     .filter(ev => ensureMatches(selector, element, ev))
+    .map((ev: Event) => {
+      if (useCapture)
+        return cloneEvent(ev, findCurrentTarget(selector, element, ev));
+
+      return ev;
+    })
     .multicast();
 }
 
+function findCurrentTarget(selector: string, element: Element, ev: Event): Element {
+  let currentTarget: Element | void = void 0;
```

Comment:
You don’t need to assign the variable `void 0`. It is defaulting to `undefined` when unassigned.